packages/create-astro/src/index.ts

```diff
@@ -337,42 +337,40 @@ export async function main() {
 		console.log(`  You can safely ignore these files, but don't delete them!`);
 		console.log(dim('  (ex: tsconfig.json, src/env.d.ts)'));
 		console.log(``);
-		tsResponse.typescript = 'default';
+		tsResponse.typescript = 'base';
 		await wait(300);
 	}
 	if (args.dryRun) {
 		ora().info(dim(`--dry-run enabled, skipping.`));
 	} else if (tsResponse.typescript) {
-		if (tsResponse.typescript !== 'default') {
```

Comment:
The diff GitHub is showing here is unnecessarily complex, the only differences from here down are:
1. Removed this line, and it's matching closing `}` (former line 375)
2. Unindented former lines 347-374 one level (as you'd expect since a wrapping block was removed)